libraries/Microsoft.Bot.Builder.Runtime/Microsoft.Bot.Builder.Runtime.csproj

```diff
@@ -0,0 +1,63 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Version Condition=" '$(ReleasePackageVersion)' == '' ">$(LocalPackageVersion)</Version>
+    <Version Condition=" '$(ReleasePackageVersion)' != '' ">$(ReleasePackageVersion)</Version>
+    <PackageVersion Condition=" '$(ReleasePackageVersion)' == '' ">$(LocalPackageVersion)</PackageVersion>
+    <PackageVersion Condition=" '$(ReleasePackageVersion)' != '' ">$(ReleasePackageVersion)</PackageVersion>
+    <Configurations>Debug;Release</Configurations>
+    <DocumentationFile>bin\$(Configuration)\netstandard2.0\Microsoft.Bot.Builder.Runtime.xml</DocumentationFile>
+  </PropertyGroup>
+    
+  <PropertyGroup>
```

Comment:
General comment for consideration: do we want to bake dependencies to LUIS, QnA, AppInsights and Azure into the runtime package? or we leave that to the project instance that composer uses? 
Consider that down the road we may want to swap LUIS by Orchestrator, a developer may want to use something different than Azure or they want to use something different than AppInishgts.

Reply:
That's a fair question. Naively, I would assume that we want to have a default set of these components provided out of the box, but you're right - the principle really ought to be that we can load these out from provided packages and incorporate them into the runtime.

It may be that we have some mechanism to represent the default packaged bundle of these which can somehow be incorporated in the same way, but for now until we get more definition around the general package story and runtime extensibility model, it felt simplest to just directly reference the SDK packages for these as we do today elsewhere.